crates/miden-lib/src/account/auth/mod.rs

```diff
@@ -53,22 +53,22 @@ impl From<RpoFalcon512> for AccountComponent {
 /// It exports the procedure `auth__tx_rpo_falcon512_procedure_acl`, which:
 /// - Checks if any of the specified auth trigger procedures were called during the transaction
 /// - If none were called, authentication is skipped
-/// - If at least one was called, performs standard RpoFalcon512 signature verification
+/// - If at least one was called, performs standard AuthRpoFalcon512 signature verification
 /// - Always increments the nonce
 ///
 /// The storage layout is:
-/// - Slot 0(value): Public key (same as RpoFalcon512)
+/// - Slot 0(value): Public key (same as AuthRpoFalcon512)
 /// - Slot 1(value): Number of trigger procedures
 /// - Slot 2(map): A map with trigger procedure roots
 ///
 /// This component supports all account types.
-pub struct RpoFalcon512ProcedureAcl {
+pub struct AuthRpoFalcon512ProcedureAcl {
```

Comment:
Shouldn't we rename this to `AuthRpoFalcon512Acl`? (i.e., drop the `Procedure` part)